### PR TITLE
Fix/recreate migrations

### PR DESCRIPTION
**Pull request by nesnoj**
Some issues occurred when updating the WAM dev to the current version since the migrations are inconsistent. Therefore, I recreated the migrations which can now be applied out of the box.

As there were a lot of model changes anyway (#10), you might want to apply the migrations first:
- Pull `fix/recreate_migrations`
- Delete all `stemp_abw_*` tables in your DB
- Reset migrations: `./manage.py migrate --fake stemp_abw zero` (you can check via `./manage.py showmigrations stemp_abw`)
- Apply migrations `./manage.py migrate stemp_abw`
- Import backup file (a recent dump `stemp_abw_db_190223_4wamdev.backup` is located in your Temp dir). make sure you do not import owners, privileges and tablespaces

### CODE REVIEW

**Comment by 4lm** [comment]
Hi @nesnoj, thanks for the info! I will have a look at it asap ...

**Comment by 4lm** [comment]
So,

TL;DR: Code looks fine. Local restore of the data to my local database worked (after some initial hickhack). I will merge this pull request.

The longer story:

Finally I could manage to migrate and restore the data to my local database. I encounterd some weird behaviour. The migration steps had to be altered. I needed 4 runs, to successfully restore the dump to my local database. Finally I realised, that after applying migrations ("./manage.py migrate stemp_abw") I have (i) to delete the newly created "stemp_abw_\*" tables in the local database again, because the restore of the dump tries to recreate them and (ii) I also have to delete all dependent sequence tables of my "stemp_abw_\*" tables as well. Which automatically should be done by "CASCADE" in the SQL snippet Ludwig gave me:

```
DO
$$
DECLARE
    row record;
BEGIN
    FOR row IN SELECT tablename FROM pg_tables WHERE schemaname = 'public' AND tablename LIKE 'stemp_abw_%'
    LOOP
        EXECUTE 'DROP TABLE IF EXISTS public.' || quote_ident(row.tablename) || ' CASCADE;';
    END LOOP;
END;
$$;
```

Weird, but it worked in the end :)

**Comment by 4lm** [comment]
Mhh,

this pull request merged to "master" and not to "dev".

"dev" is now 9 commits behind "master".

Shall I: 

- merge "master" to "dev"?
OR
- this branch ("fix/recreate migrations") to dev?

We are still working from "dev" and not "master" - right?

**Comment by nesnoj** [comment]
You're right, sorry for the confusion, I updated dev accordingly.
Yes, go on in dev..

**Comment by 4lm** [comment]
Perfect, thx a lot!
